build-node-4.2.4.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+VERSION="v4.2.4"
+
+ARCHES=(
+  linux-x86
+  linux-x64
+  darwin-x64
+)
+
+DESTINATION_DIRECTORIES=(
```

Comment:
A comment indicating that these two lists need to be the same length would be helpful. Alternatively, doing a "tuple" thing via string splitting might be better.